### PR TITLE
Undocument allow-drop-table property

### PR DESCRIPTION
**Pull request by Jessie212**

### CODE REVIEW

File: docs/src/main/sphinx/connector/cassandra.rst [hunk 1]
```diff
@@ -62,9 +62,6 @@ Property Name                                      Description
                                                    ``EACH_QUORUM``, ``QUORUM``, ``LOCAL_QUORUM``, ``ONE``, ``TWO``,
                                                    ``THREE``, ``LOCAL_ONE``, ``ANY``, ``SERIAL``, ``LOCAL_SERIAL``.
 
-``cassandra.allow-drop-table``                     Set to ``true`` to allow dropping Cassandra tables from Trino
```
Comment (hashhar) [inline docs/src/main/sphinx/connector/cassandra.rst:65]: I think there is some confusion. Only JDBC based connectors will be affected by what @electrum mentioned about `allow-drop-table`. So Cassandra would still need this config most probably.

Also, let's not remove things from documentation until the feature actually gets implemented.

**Comment by findepi** [comment]
Update the commit title. "Remove allow-drop-table property" can be read as "allow-drop-table property has been removed", while this changes docs, not the actual properties.

**Comment by Jessie212** [comment]
@findepi Okay. Good call on the commit title.

**Comment by mosabua** [comment]
I am closing this for now since we are waiting for engineering work

**Pull request closed without merging**